formicaio/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1
+id: formicaio
+name: Formicaio
+tagline: Il potere delle formiche
+icon: ""
+category: social
+version: "0.3.0"
+port: 52100
+description: >-
```

Comment:
Maybe you could add some more details about what users need to do for port-forwarding and so on.

And in general more details would not hurt. Still kind of unclear what the app does exactly.